Apromore-Clients/manager-client/src/main/java/org/apromore/manager/client/ManagerServiceImpl.java

```diff
@@ -110,10 +113,50 @@
 
     private boolean isGEDMatrixReady = true;
     
-    private static final Logger LOGGER = LoggerFactory.getLogger(ManagerServiceImpl.class);
+    private final Logger logger = LoggerFactory.getLogger(this.getClass());
 
+// Implementation of ManagerService
 
-    // Implementation of ManagerService
+    /**
+     * Logout, in Keycloak, user sessions associated with said user.
+     *
+     * @param username The username of the user to logout.
+     *
+     * @return <code>true</code> if the user was logged-out, <code>false</code> otherwise.
+     */
+    @Override
+    public boolean logoutUserAllSessions(final String username) throws Exception {
+        Boolean restRespResult = false;
+
+        final RestTemplate restTemplate = new RestTemplate();
+
+        try {
+            restRespResult =
+                    restTemplate.getForObject("http://securityms.apromoresso.net:8282/logout/" + username, Boolean.class);
+            logger.debug("\n\nrestRespResult: {}", restRespResult);
+        } catch (final Exception e) {
+            final String exceptionMsg = e.getMessage();
+
+            if (e instanceof CertificateException || ((exceptionMsg != null) &&
+                    (exceptionMsg.indexOf("No subject alternative DNS") != -1)) ) {
+                logger.info("This is a non-fatal exception {}; can continue", e.getMessage());
+
+                restRespResult =
+                        restTemplate.getForObject("https://localhost:8143/logout/" + username, Boolean.class);
+                logger.debug("\n\nrestRespResult: {}", restRespResult);
+
+                restRespResult = true;
+            } else {
+                e.printStackTrace();
```

Comment:
Send logging to the system logger (e.g. with SLF4J) rather than writing it to System.out/err.

Reply:
Utilised ExceptionUtils.getStackTrace() per Nolan's/code review suggestion [DONE]